### PR TITLE
API methods for returning key/value type data for census

### PR DESCRIPTION
**Pull request by LinuxBozo**
For use in local database (LevelDB/IndexedDB)

### CODE REVIEW

**Comment by LinuxBozo** [comment]
Noticed I'm missing a bit in the aggregate..

**Comment by LinuxBozo** [comment]
Still not quite right for selection by year..

File: controllers/localdb/census/index.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+    /**
+     * @param {String} activityYear
+     * @return {json}
+     */
+    router.get('/msaCodes/:activityYear', function(req, res) {
+        var keyParams = ['msa_code'];
+        var value = 'msa_name';
+
+        CensusService.getKeyValueData(req.params.activityYear, keyParams, value, function(err, result) {
```
Comment (doelleri) [inline controllers/localdb/census/index.js]: Looks like this block is the same in each method. Refactor?

File: services/CensusService.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 /*jshint maxparams: 6 */
 'use strict';
 
-var Census = require('../models/census');
-
-var exists = require('../lib/queryUtil').exists;
+var Census = require('../models/census'),
+    queryUtil = require('../lib/queryUtil'),
+    _ = require('underscore');
```
Comment (doelleri) [inline services/CensusService.js]: Doesn't look like you use `_` in here after refactoring to `queryUtil`.

**Pull request merged**